src/components/buttons/ButtonAdd.js

```diff
@@ -0,0 +1,36 @@
+import { useDispatch } from 'react-redux';
+import { useState, useEffect } from 'react';
+import { addBook } from '../../redux/books/booksSlice';
+import books from '../ListBooks';
+
+const ButtonAdd = () => {
+  const dispatch = useDispatch();
+  const [book, setBook] = useState('');
+  const [author, setAuthor] = useState('');
+
+  const SubmitBook = (e) => {
+    const button = e.target;
+    const dataForm = button.parentElement;
+
+    setBook(dataForm.bookTitle.value);
+    setAuthor(dataForm.author.value);
+    if (book === '' || author === '') {
+      document.querySelector('.messagecont').textContent = 'Enter all the data please';
+    }
+  };
+
+  useEffect(() => {
+    if (book !== '' && author !== '') {
+      dispatch(addBook(books(book, author)));
+      setBook('');
+      setAuthor('');
+      document.querySelector('form').reset();
+      document.querySelector('.messagecont').textContent = '';
+    }
+  }, [book, author, dispatch]);
```

Comment:
- [ ] [OPTIONAL] You can also add category as a drop down input field.
<img width="923" alt="253909755-cc4aa165-db0b-4d8a-9c20-ee6bf947db5f" src="https://github.com/AndreaM2429/Bookstore/assets/41163571/b83d4979-9255-4755-882b-5ebe57dc8db8">